src/components/dropdown-menu/DropdownMenuItemBase.tsx

```diff
@@ -60,39 +60,50 @@ export function DropdownMenuItemBase({
 
   if (isCheckable) {
     checkMarkOrPlaceHolder = isChecked ? (
-      <IconCheck color="echoes-color-icon-selected" />
+      <StyledIconCheck color="echoes-color-icon-selected" />
     ) : (
       <CheckmarkPlaceholder />
     );
   }
 
   return (
-    <StyledDropdownMenuItemBase
+    <StyledRadixDropdownMenuItem
       aria-label={ariaLabel}
       className={className}
       disabled={isDisabled}
       onClick={isDisabled ? undefined : onClick}>
       <StyledLeftHandSide>
         {checkMarkOrPlaceHolder}
 
-        {prefix}
+        {prefix && <StyledPrefix>{prefix}</StyledPrefix>}
 
         <StyledLabelAndHelpText>
-          {children}
+          <StyledLabel>{children}</StyledLabel>
 
           <StyledHelpText isDisabled={isDisabled}>{helpText}</StyledHelpText>
         </StyledLabelAndHelpText>
       </StyledLeftHandSide>
 
       <StyledSuffix>{suffix}</StyledSuffix>
-    </StyledDropdownMenuItemBase>
+    </StyledRadixDropdownMenuItem>
   );
 }
 
+const StyledIconCheck = styled(IconCheck)`
+  font-size: var(--echoes-font-size-body-medium);
+`;
+
 const CheckmarkPlaceholder = styled(IconCheck)`
+  font-size: var(--echoes-font-size-body-medium);
   visibility: hidden;
 `;
```

Comment:
NP: you could reuse the StyledIconCheck instead, so that any changes to it doesn't need to be reproduced here.